project-minimization-plugin/src/main/kotlin/org/plan/research/minimization/plugin/settings/MinimizationPluginSettings.kt

```diff
@@ -6,4 +6,4 @@ import com.intellij.openapi.components.State
 
 @Service(Service.Level.PROJECT)
 @State(name = "MinimizationPluginSettings")
-class MinimizationPluginSettings : SimplePersistentStateComponent<MinimizationPluginState>(MinimizationPluginState())
+class MinimizationPluginSettings : SimplePersistentStateComponent<MinimizationPluginStateObservable>(MinimizationPluginStateObservable())
```

Comment:
That shouldn't be changed, but you can add an additional field called `stateObservable`